src/js/iqeEnablement.js

```diff
@@ -1,41 +1,39 @@
 let results = [];
-let initted = false;
 
 function init () {
+    window.localStorage.setItem('iqe:chrome:init', 'true');
     console.log('[iqe] initialized'); // eslint-disable-line no-console
 
     const open = window.XMLHttpRequest.prototype.open;
     const send = window.XMLHttpRequest.prototype.send;
 
     // must use function here because arrows dont "this" like functions
-    window.XMLHttpRequest.prototype.open = function openReplacement(method, url) { // eslint-disable-line func-names
```

Comment:
I want to see the function names in a stack trace, not anonymous anonymous anonymous.

I think we had the opposite style rule before because this was such an issue.

Reply:
OK - will fix that!

Reply:
Oh I see, okay. Good point.